activity/src/main/java/ru/fors/activity/data/controller/ActivityController.kt

```diff
@@ -49,4 +51,9 @@ class ActivityController(
     fun delete(@PathVariable id: Long) {
         deleteActivityUseCase.execute(id)
     }
+
+    @PatchMapping("/close/{id}")
```

Comment:
```suggestion
    @PostMapping("/{id}/close")
```
ну вообще мапинг должен быть наоборот. Patch для изменения только сущности, а тут меняется состояние системы